scripts/src/main/resources/scripts/command/mvn

```diff
@@ -306,7 +306,7 @@ function doCheckNoSnapshots() {
 
 function doRunBuild() {
   local maven_cmd="${MVN}"
-  local maven_arguments=""
+  local maven_arguments="--settings=${DEVON_IDE_HOME}/conf/.m2/settings.xml"
```

Comment:
dont we need to export this to `MAVEN_ARGS` via `devon.properties?
This only applies when `mvn` is called via `devon` as wrapper command.
However, I see this bug occurring all the time without `devon` command being involved as wrapper but only to setup env variables.
My assumption is that the actual bug #706 somehow originates from m2e eclipse plugin and happens since a newer version of that plugin so it did not happen before and suddenly came with some ide or eclipse update...

Reply:
The current "hack" to relocate the settings to this directory happens here:
https://github.com/devonfw/ide/blob/master/scripts/src/main/resources/scripts/devon.properties#L19

So far this worked well for commandline (with and without `devon` wrapper) as well as for all supported IDEs.

Reply:
later down the `maven_arguments` are overridden (`maven_arguments="${MAVEN_ARGS}"`) so with maven 3.x your change will have no effect. As maven 4.x is still not released, this PR does not work or make a big difference.
Should we close this or should we change the later code in this PR to honor existing `maven_arguments` and append to it instead of overriding.